incubator/buzzfeed-sso/values.yaml

```diff
@@ -0,0 +1,98 @@
+# Default values for buzzfeed-sso.
+# Commented values are required for SSO to work
+
+# emailDomain: 'email.mydomain.foo'
+# rootDomain: 'mydomain.foo'
+
+auth:
+  # domain: "sso-auth.mydomain.foo"
+  replicaCount: 1
+  resources:
+    limits:
+      memory: "256Mi"
+      cpu: "200m"
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+  service:
+    type: ClusterIP
+    port: 80
+  # Generate these secrets with the command:
+  # 'openssl rand -base64 32 | head -c 32 | base64'
+  secret: {}
+    # codeSecret: ''
+    # cookieSecret: ''
+  # # Or if you do not want to create the secret via helm
+  # customSecret: my-sso-auth-secret
+  tls:
+    secretName: sso-auth-tls-secret
+
+proxy:
+  # virtualHost: "*.mydomain.foo"
+  # providerUrlInternal: https://sso-auth.mydomain.com
+  cluster: dev
+  replicaCount: 1
+  resources:
+    limits:
+      memory: "256Mi"
+      cpu: "200m"
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+  service:
+    type: ClusterIP
+    port: 80
+  # Generate these secrets with the command:
+  # 'openssl rand -base64 32 | head -c 32 | base64'
+  secret: {}
+    # clientId: ''
+    # clientSecret: ''
+    # cookieSecret: ''
+  # # Or if you do not want to create the secret via helm
+  # customSecret: my-sso-proxy-secret
+
+provider:
+  google: {}
+  # google:
+  #   adminEmail: me@mydomain.foo
+  #   secret:
+  #     clientId: foo123123-fake123123.apps.googleusercontent.com
+  #     clientSecret: googleOauthClientSecret
+  #     serviceAccount: 'service account content in JSON format'
+  # # Or if you do not want to create the secret via helm
+  # google:
+  #   adminEmail: me@mydomain.foo
+  #   customSecret: my-sso-google-secret
+
+image:
+  repository: buzzfeed/sso
+  tag: v1.1.0
+  pullPolicy: IfNotPresent
+
+ingress:
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # certmanager.k8s.io/cluster-issuer: my-letsencrypt-issuer
+    # ingress.kubernetes.io/ssl-redirect: "true"
+  hosts: []
+  #  - domain: mybackend.mydomain.foo
+  #  - path: /
```

Comment:
I think the `-` should be removed here, no?

Reply:
@darioblanco  If I'm not mistaken, the `hosts` field takes an array of objects that each contains `domain` and `path`, so it should look like this:
```
hosts:
  - domain: xxx
    path: xxxx
```

In the code above, `hosts` is an array of two elements, one containing `domain`, the other `path`, was that really on purpose?

Reply:
Ohhh now I get it. Even if I looked at it more than once I did not realize the mistake. Will fix it right away in the comments so is not misleading